src/components/tasks/BaseTaskOverlay.js

```diff
@@ -0,0 +1,367 @@
+import {useLocation, useNavigate, useParams} from "react-router-dom";
+import {useTasks} from "../../hooks/useDataContext";
+import {useContactList} from "../../hooks/useContactList";
+import React, {useState} from "react";
+import {getStatusFromIndex} from "../../tools/status";
+import {ReactSVG} from "react-svg";
+import {handleColorInjection, handleRotateInjection, handleSizeInjection} from "../../tools/svg";
+import UnderlineIconInput from "../base/UnderlineIconInput";
+import NameIcon from "../icons/NameIcon";
+import Separator from "../base/Separator";
+
+const BaseTaskOverlay = (
+    {taskId, onExit, name = "", description = "",date = "", selectedPriority = "", taskCategory = "Select task category",
+        assigneesArr = [], subtasksArr = []}) => {
```

Comment:
🏴‍☠️ 🏴‍☠️ Arr, arr, arr ! 🏴‍☠️ 🏴‍☠️ 
There is a convention for naming: any variable finishing with an `s` is automatically an array.